ISISNeutronMuon/MDANSE
# Add qprocess job launching

**Description of work**
We switch from QThread to multiprocessing.Process for starting new jobs. This way the jobs will not be competing for the Python GIL.

**Fixes**
A State pattern has been implemented for jobs.
A context menu lets the user pause, restart and terminate jobs from the GUI.
Each job is a process and uses pipes to communicate with the main process.

**To test**
Run a few longer jobs at the same time. See if they all use 100% of a CPU core. Try pausing and resuming them.
